lib/mainscreen.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
```

Comment:
don't name it mainscreen.dart

call it `main.dart` instead and better if you add it inside `screens` folder